app/routes/surveys/components/SurveyEditor/SurveyEditor.tsx

```diff
@@ -64,21 +60,22 @@ function TemplateTypeDropdownItems({
 
   return (
     <Dropdown.List>
-      {Object.keys(EVENT_CONSTANTS).map((eventType: EventType) => (
-        <Dropdown.ListItem key={eventType}>
-          <Link
-            to="#"
-            onClick={(e) => {
-              e.preventDefault();
-              e.stopPropagation();
-              destroy();
-              push(link(eventType));
-            }}
-          >
-            {eventTypeToString(eventType)}
-          </Link>
-        </Dropdown.ListItem>
-      ))}
+      {(Object.keys(EVENT_CONSTANTS) as (keyof typeof EVENT_CONSTANTS)[]).map(
```

Comment:
```suggestion
      {(Object.keys(EVENT_CONSTANTS) as EventType[]).map(
```
Is this not the same?

Reply:
They should be equal, but it wouldn't actually check that the keys are the same strings as EventType